packages/phlo-observatory/src/phlo_observatory/src/v2/design/tokens.css

```
@@ -16,6 +16,8 @@
     linear-gradient(180deg, oklch(0.995 0.003 286 / 0.72), transparent 18rem),
     var(--v2-canvas);
   color: var(--v2-text);
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.11.1)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)